build.sbt

```diff
@@ -1,96 +1,36 @@
-import sbtcrossproject.CrossPlugin.autoImport.crossProject
-
-inThisBuild(Seq(
-  name := "Scala DOM Types",
-  normalizedName := "domtypes",
```

Comment:
Where does the new config take `normalizedName` from? I'd prefer for it to be explicit.

Reply:
sbt says it's "not used by any other settings/tasks", so it doesn't affect anything